pages/admin/personen/details/Zuordnungen.page.ts

```diff
@@ -0,0 +1,147 @@
+import { expect, Page } from '@playwright/test';
+import { Autocomplete } from '../../../../elements/Autocomplete';
+import { AddZuordnungWorkflowPage } from './zuordnung-workflows/AddZuordnungWorkflow.page';
+import { BefristungWorkflowPage } from './zuordnung-workflows/BefristungWorkflow.page';
+
+export interface ZuordnungCreationParams {
+  rolle: string;
+  organisation?: string;
+  prefilledOrganisation?: string;
+  kopers?: string;
+  befristung?: string;
+}
+export interface ZuordnungValidationParams {
+  organisation: string; dstNr?: string; rolle?: string; befristung?: string; status?: 'unchanged' | 'delete' | 'create';
+}
+
+export class ZuordnungenPage {
+  public constructor(private readonly page: Page) {}
```

Comment:
```suggestion
constructor(
  private readonly page: Page,
  private readonly addZuordnungWorkflowFactory: (page: Page) => AddZuordnungWorkflowPage = p => new AddZuordnungWorkflowPage(p),
  private readonly befristungWorkflowFactory: (page: Page) => BefristungWorkflowPage = p => new BefristungWorkflowPage(p)
) {}
```
you could inject dependencies like AddZuordnungWorkflowPage and BefristungWorkflowPage factories instead of instantiating them directly.